pkg/registry/apis/dashboard/legacy/testdata/mysql--query_dashboards-history_uid.sql

```diff
@@ -8,12 +8,14 @@ SELECT
   provisioning.updated     as repo_ts,
   dashboard.created, created_user.uid as created_by, dashboard.created_by   as created_by_id,
   dashboard.updated, updated_user.uid as updated_by, dashboard.updated_by   as updated_by_id,
-  dashboard.version, '' as message, dashboard.data
+  dashboard.version, '' as message, 
+  dashboard.data, dashboard.api_version
 FROM `grafana`.`dashboard` as dashboard
 LEFT OUTER JOIN `grafana`.`dashboard_provisioning` as provisioning ON dashboard.id = provisioning.dashboard_id
 LEFT OUTER JOIN `grafana`.`user` as created_user ON dashboard.created_by = created_user.id
 LEFT OUTER JOIN `grafana`.`user` as updated_user ON dashboard.updated_by = updated_user.id
 WHERE dashboard.is_folder = FALSE
   AND dashboard.org_id = 2
     AND dashboard.uid = 'UUU'
+    AND dashboard.deleted IS NULL
```

Comment:
Is this expected? Tested dashboard history & things are working as expected with `kubernetesClientDashboardsFolders`, but this file name makes me think it shouldn't have this in the query?

Reply:
without it, soft deleted dashboards show up in the list 😬

Reply:
ahhh okay makes sense, thanks!